### ISSUE TITLE
Installing global npm packages needs sudo

### ISSUE THREAD

**Issue by webmaster128** [ArkEcosystem/deployer#25]
Some node packages need sudo to be installed globally, e.g. forever

```
$ npm install -g forever
npm WARN checkPermissions Missing write access to /usr/local/lib
[...]
```

but in https://github.com/ArkEcosystem/ark-deployer/blob/master/lib/dependencies.sh#L63 they are tried to be installed without sudo

**Comment by alexbarnsley** [comment]
@webmaster128 how was your npm installed? Did you use the nvm command from the readme? Also, what OS and version are you trying to install the deployer with? Thanks

**Comment by webmaster128** [comment]
Hi @alexbarnsley, thanks for feedback.

I am installing on Ubuntu 18.04 Desktop (which works after resolving some minor issues, so I'd love if you would be willing to support it). I am not a fan of running random bash scripts from the Internet (nvm) and thus installed node and npm via the package manager `sudo apt-get install nodejs npm`.

The current versions of node and npm are
```
$ node --version
v8.10.0
$ npm --version
3.5.2
```

But maybe we can avoid installing global node modules at all and just run for example `./node_modules/forever/bin/forever`?

**Comment by alexbarnsley** [comment]
That's fair enough regarding random scripts. It is from the [nvm repo](https://github.com/creationix/nvm) but I can understand how that might look to some. When I've got a bit of free time, I will see if I can change that. 

Currently Ubuntu 16 is advised in the [node setup guide](https://blog.ark.io/how-to-setup-a-node-for-ark-and-a-basic-cheat-sheet-4f82910719da). This is specific to ark-node. It may work on others but hasn't been thoroughly tested (I've since added this as a prerequisite to the [README](https://github.com/ArkEcosystem/ark-deployer/blob/master/README.md)). For that reason, the same applies for deployer - it's not been extensively tested on any other OS. 

I don't think I will do anything further with this (but will keep it open), as the deployer is likely to change a lot when v2 of core comes out, meaning a lot may get rewritten, or global packages may need installing. This will all become clear to me soon I think.
